internal/services/v1/permissions.go

```diff
@@ -45,6 +50,8 @@ func (ps *permissionServer) CheckPermission(ctx context.Context, req *v1.CheckPe
 			Relation:  normalizeSubjectRelation(req.Subject),
 		},
 	})
+	metaHandler.SetFor(cr.Metadata)
+	shared.UpdateServiceMetrics("v1.CheckPermission", cr.Metadata)
```

Comment:
Should we be counting/returning these on read/write/delete operations as well? At least read?

Reply:
Right now, they'd all be zero, since there are no dispatches involved. Maybe a different set of metrics?

Reply:
We can handle this in a follow-up.